desktop/src/main/java/bisq/desktop/main/dao/wallet/BsqBalanceUtil.java

```diff
@@ -125,6 +136,9 @@ public void onUpdateBalances(Coin availableBalance,
         availableNonBsqBalanceTextField.setVisible(isNonBsqBalanceAvailable);
         availableNonBsqBalanceTextField.setManaged(isNonBsqBalanceAvailable);
         availableNonBsqBalanceTextField.setText(bsqFormatter.formatBTCWithCode(availableNonBsqBalance.value));
+        
+        // Update reputationBalanceTextField
+        reputationBalanceTextField.setText(bsqFormatter.formatBSQSatoshisWithCode(myBlindVoteListService.getCurrentlyAvailableMerit())); 
```

Comment:
I think it would be more appropriate to use DaoFacade.getAvailableMerit() here but perhaps not super important

Reply:
> Still need ack from @ManfredKarrer for changes in DAO code to be able to merge

@sqrrm Do you think we need the ACK from @ManfredKarrer that for display changes as well or just for DAO changes?

Reply:
> I think it would be more appropriate to use DaoFacade.getAvailableMerit() here but perhaps not super important

OK. I will update.